src/Utils/utils.ts

```
@@ -544,3 +544,17 @@ export const fahrenheitToCelsius = (fahrenheit: number) => {
 };
+
+export const handleNegativeValue = (
+  event: React.FormEvent<HTMLInputElement>,
+): void => {
+  const input = event.currentTarget;
+
+  // Remove all instances of '-' from the input value
+  input.value = input.value.replace(/-/g, "");
+
+  // Optional: Ensure the input value is a valid number
+  if (isNaN(Number(input.value))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)